modules/monitoring-adding-cluster-id-labels-to-metrics.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="adding-cluster-id-labels-to-metrics_{context}"]
+= Adding cluster ID labels to metrics
+
+If you manage multiple {product-title} clusters and use the remote write feature to send metrics data from these clusters to an external storage location, you can add cluster ID labels to identify the metrics data coming from different clusters.
+
+You add cluster ID labels to metrics by configuring the write relabel settings for remote write storage. 
```

Comment:
There seem to be several actions in this second paragraph:

1. Configure the write relabel settings for remote write storage. 
2. Add cluster ID labels to the metrics. 
3. Query these labels to identify the source cluster for a metric and distinguish that metric data from similar metric data sent by other clusters. 

Could this content be turned into a small procedure? 

What do you think of this structure?: 

Abstract:

If you manage multiple {product-title} clusters and use the remote write feature to send metrics data from these clusters to an external storage location, you can add cluster ID labels to identify the metrics data coming from different clusters. This way, when you manage many clusters for multiple customers and send metric data to a single centralized storage system, you can use cluster ID labels to query metrics for a particular cluster or customer.

.Procedure

1. Configure the write relabel settings for remote write storage. 
2. Add cluster ID labels to the metrics. 
3. Query these labels to identify the source cluster for a metric and distinguish that metric data from similar metric data sent by other clusters. 

.Additional resources 

- Creating Cluster ID labels for metrics<link>

Reply:
Note, I'm not saying this content _needs_ to be changed according to modularization principles. 

It simply seems to me that you have an intro/abstract followed by a series of actions, which to me, signifies a procedure module. 

What do you think?

Reply:
This is a concept module, and the procedure comes next in a different file, but I like the idea of including a conceptual overview of the overall procedure here.  I also like your rewrite of the conceptual content and will use it to revise. I'll try a revision and ping you to have a look.